cfpb/consumerfinance.gov
# Update email-signup.html - Remove inline right margin

@niqjohnson @willbarton one of you authored this in https://github.com/cfpb/consumerfinance.gov/pull/7107/commits/f22175f0f577a291824c4f575a144bb161fad544

This code adds a right-margin to the sign up button on pages like https://www.consumerfinance.gov/an-essential-guide-to-building-an-emergency-fund/

<img width="378" alt="Screen Shot 2023-02-24 at 11 58 55 AM" src="https://user-images.githubusercontent.com/704760/221240727-1020c5f5-3934-404f-9b2b-64931d485d86.png">

It looks like this code would be necessary if there was a button or link to the right of the sign up button, but looking at the code there is only one button. 

## Removals

- Remove inline right margin from sign up button on button-only email signup.

## How to test this PR

1. Check the email signup at pages like https://www.consumerfinance.gov/an-essential-guide-to-building-an-emergency-fund/ and see that there's no issue with the button not having a right margin.
